ek9000App/src/devEK9000.h

```diff
@@ -205,9 +203,9 @@ class devEK9000 {
 	/* The driver */
 	drvModbusAsyn* m_driver;
 
-	char* m_name;
-	char* m_portName;
-	char* m_ip;
+	std::string m_name;
```

Comment:
Switching to `std::string` - so much free/duplicate boilerplate went away. Nice!